roles/openshift_manage_node/tasks/main.yml

```diff
@@ -59,19 +59,29 @@
   delegate_to: "{{ openshift_master_host }}"
 
 - name: Set node schedulability
-  command: >
-    {{ hostvars[openshift_master_host].openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable={{ 'true' if openshift.node.schedulable | bool else 'false' }}
-    --config={{ openshift_manage_node_kubeconfig }}
-    -n default
-  when: "'nodename' in openshift.node"
+  oadm_manage_node:
+    node: "{{ openshift.node.nodename | lower }}"
+    schedulable: "{{ 'true' if openshift.node.schedulable | bool else 'false' }}"
+    config: "{{ openshift_manage_node_kubeconfig }}"
```

Comment:
set to ``kubeconfig: "{{ openshift.common.config_base }}/master/admin.kubeconfig"`` in @mtnbikenc 's PR